.circleci/config.yml

```diff
@@ -1,359 +1,13 @@
----
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
-
+# Use a package of configuration called an orb.
 orbs:
-  lighthouse-check: foo-software/lighthouse-check@0.0.8
-
+  # Declare a dependency on the welcome-orb
+  welcome: circleci/welcome-orb@0.4.1
+# Orchestrate or schedule a set of jobs
 workflows:
-  main:
+  # Name the workflow "welcome"
+  welcome:
+    # Run the welcome/run job in its own container
     jobs:
```

Comment:
# Name the workflow "welcome"
  welcome:
    # Run the welcome/run job in its own container
    jobs: